packages/office-ui-fabric-react/src/components/ComboBox/ComboBox.types.ts

```diff
@@ -118,6 +119,11 @@ export interface IComboBoxProps extends ISelectableDroppableTextProps<IComboBox>
    */
   buttonIconProps?: IIconProps;
 
+  /**
+   * The AutofillProps to be passed into the Autofill component inside combobox
+   */
+  autofillProps?: IAutofillProps;

```

Comment:
Could this be a slot called `autofill`? If so, we could change this prop name to `autofill` allowing it to expand to a full slot later as a backwards compatible API change.

Reply:
I thought about that, but existentially, what is a combobox if the input field was something else?

Reply:
Hmm, I guess another way of asking that question is do we want to restrict the `autofill` slot to props only because we can't think of a valid use case of replacing it? What if someone wanted to provide their own `Autofill` component? What do we lose by making it a full slot?

Reply:
BTW, I just kind of followed the pattern of the props above:

```ts
  /**
   * The IconProps to use for the button aspect of the combobox
   */
  buttonIconProps?: IIconProps;

  /**
   * The AutofillProps to be passed into the Autofill component inside combobox
   */
  autofillProps?: IAutofillProps;
```

However, I realize that this is a new prop. So I'll just rename this to `autofill` and go with it knowing that the inconsistency will go away in 7.0 w/ slots.